habitat-baselines/habitat_baselines/rl/hrl/skills/skill.py

```diff
@@ -137,37 +151,37 @@ def _apply_postcond(
         apply_action.set_param_values(entities)
 
         log_info[env_i]["pddl_action"] = apply_action.compact_str
-        return actions
+        return actions[idx]
 
     def should_terminate(
         self,
-        observations,
-        rnn_hidden_states,
-        prev_actions,
-        masks,
-        actions,
-        hl_says_term,
+        observations: Observations,
+        rnn_hidden_states: torch.Tensor,
+        prev_actions: torch.Tensor,
+        masks: torch.Tensor,
+        actions: torch.Tensor,
+        hl_policy_wants_termination: torch.BoolTensor,
```

Comment:
is this termination of the low-level skill, or the episode?

Reply:
It is for the low-level skill. I've updated the name to reflect this.